Bonsai/Program.cs

```diff
@@ -86,6 +86,7 @@ internal static int Main(string[] args)
             });
             parser.Parse(args);
 
+            launchEditor &= !exportImage;
```

Comment:
While I do love a nice one-liners, IMO this is more readable at a glance:

```csharp
if (exportImage)
    launchEditor = false;
```

Reply:
@PathogenDavid yeah, I did toggle between the two forms, I was wondering in the future we may have more flags which cancel editor launch but I guess they could all go under the same if statement.

Will come back to this after mulling it over.